website/docs/r/x509_authentication_database_user.html.markdown

```diff
@@ -17,6 +17,8 @@ description: |-
 
 -> **NOTE:** Groups and projects are synonymous terms. You may find group_id in the official documentation.
 
+-> **NOTE:** Before provider version 1.14.0, Self-managed X.509 Authentication was disabled for the project when this resource was deleted. From that version on, it is not disabled because the other users may want to continue using X.509.
+
```

Comment:
```suggestion
-> **NOTE:** Before provider version 1.14.0, Self-managed X.509 Authentication was disabled for the project when this resource was deleted. From that version on, it won't be disabled to let other users continue using X.509 within the same project.

```

@lantoli check if `within the same project` is correct please. If you are ok, accept the suggestion (just a minor word refactoring).

Reply:
thx, changing for a similar sentence